### PR TITLE
Howdy section - including navbar & howdy top section

### PR DESCRIPTION
**Pull request by snack-attack**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,16 +6,85 @@
...
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        <link rel="stylesheet" href="">
+        <link rel="stylesheet" href="dist/css/styles.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
     </head>
     <body>
         <!--[if lt IE 7]>
             <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
+        <div class="navbar">
```
Comment (alicemcneil) [inline index.html]: would you add an in-page navbar or something like a "back to top" link?

File: index.html [hunk 2]
```diff
@@ -80,7 +80,7 @@ <h1>Bath Institute for Rheumatic Disease</h1>
                 </div>
                 <div class="work-card">
                     <div class="meta">
-                        <img src="assets/img/5.jpg" class="photo"></img>
+                        <img src="assets/img/33.jpg" class="photo"></img>
```
Comment (alicemcneil) [inline index.html:83]: i couldn't see your portfolio images?

File: css/styles.css [hunk 3]
```diff
@@ -194,32 +202,28 @@ nav ul li a {
 }
 
 .work-card:hover .photo {
-    -webkit-transform: scale(1.3) rotate(3deg);
-            transform: scale(1.3) rotate(3deg);
+    -webkit-transform: scale(1.1) rotate(3deg);
+            transform: scale(1.1) rotate(3deg);
```
Comment (alicemcneil) [inline css/styles.css:206]: i couldn't see this transform, the images were missing

File: css/styles.css [hunk 4]
```diff
@@ -232,21 +236,25 @@ nav ul li a {
...
     text-align: right;
 }
+
 .work-card .description .read-more a {
     color: #FC5956;
     display: inline-block;
     position: relative;
 }
+
 .work-card .description .read-more a:after {
```
Comment (alicemcneil) [inline css/styles.css]: love this!

Comment (alicemcneil) [inline css/styles.css:258]: love this little link square thing!

File: css/styles.css [hunk 5]
```diff
@@ -232,21 +236,25 @@ nav ul li a {
     margin: 0;
     font-size: 1.7rem;
 }
+
 .work-card .description h2 {
```
Comment (alicemcneil) [inline css/styles.css:240]: i wonder if this font is totally legible this small?/ maybe try a little less bold or larger text?

File: css/styles.css [hunk 6]
```diff
@@ -5,18 +5,14 @@
     margin: 0;
 }
```
Comment (alicemcneil) [inline css/styles.css:7]: love your page hun; the design is really suave (and responsive!)

File: css/styles.css [hunk 7]
```diff
@@ -65,32 +61,40 @@ a {
...
 
+i {
+    font-size: 1.5rem; 
+}
+
+.dot {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
```
Comment (alicemcneil) [inline css/styles.css:73]: maybe you could have a navbar or home button that stays? or even a back to top link to save the scroll? :)

File: css/styles.css (hunk 6 above)
Comment (alicemcneil) [reply css/styles.css:7]: also theres a worrying lack of cowboy hats... **pls sort this out ASAP**

File: index.html (hunk 1 above)
Comment (snack-attack) [reply index.html]: I agree, navbar should be renamed to socials-links or something more semantic! Good catch.

File: index.html (hunk 2 above)
Comment (snack-attack) [reply index.html:83]: i'll make sure this is working in the work branch :)

File: css/styles.css (hunk 5 above)
Comment (snack-attack) [reply css/styles.css:240]: 👍

File: css/styles.css (hunk 7 above)
Comment (snack-attack) [reply css/styles.css:73]: Maybe one day!!! I'll add that to the roadmap 👍

**Pull request closed without merging**